### DISCUSSION TITLE
Using USB D-/D+ as general IO?

### DISCUSSION THREAD

**Discussion by klauweg** [micropython/micropython#14226] [ESP32]
Is it possible to use Pin 19/20 as general IO Pins with Micropython on ESP32-S3?
Maybe i have to burn efuses?
Or can this configured on runtime?
Apparently it is somehow related to jtag also.

**Comment by Lobo-T** [comment]
I think you have to compile without the switches in ports/esp32/boards/sdkconfig.usb.  But I have not tried it myself.
